fets/model/mlcube/mlcube.yaml

```
@@ -0,0 +1,25 @@
+name: MLCommons FeTS
+description: MLCommons FeTS integration
+authors: 
+ - {name: "MLCommons Best Practices Working Group"}
+ - {name: "Maximilian Zenk (DKFZ)"}
+
+platform:
+  accelerator_count: 0
+
+docker:
+  # Image name.
+  image: mlcommons/fets_model:0.0.1
+  # image: docker.synapse.org/syn28561391/test_mlcube:latest
+  # Docker build context relative to $MLCUBE_ROOT. Default is `build`.
+  build_context: "../project"
+  # Docker file name within docker build context, default is `Dockerfile`.
+  build_file: "Dockerfile"
+
```

Also regarding GPUs:
```suggestion
  gpu_args: "--gpus all"   # passed to docker run if platform.accelerator_count > 0

```

---

I added a better description for this, thanks.